pycloudlib/azure/cloud.py

```diff
@@ -452,21 +459,27 @@ def delete_image(self, image_id):
                 "Error deleting %s. Request returned %d", image_id, resp_code
             )
 
+    def _get_image(self, release, image_map):
+        release = image_map.get(release)
+        if release is None:
+            msg = "No Ubuntu image found for {}. Expected one of: {}"
+            raise ValueError(
+                msg.format(release, " ".join(UBUNTU_DAILY_IMAGES.keys()))
```

Comment:
We should use image_map.keys() here because UBUNTU_DAILY_IMAGES and UBUNTU_RELEASED_IMAGES will frequently differ in keys.
```suggestion
                msg.format(release, " ".join(image_map.keys()))
```